Backend/OnboardingInsuranceAPI/Services/DataContext.cs

```diff
@@ -14,17 +14,19 @@ public class DataContext : DbContext
     public DbSet<UserInfo> Users { get; set; }
 
     protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
-        => optionsBuilder.UseCosmos(
-            Environment.GetEnvironmentVariable("AccountEndpoint"),
-            Environment.GetEnvironmentVariable("AccountKey"),
-            databaseName: "Data",
-            options =>
-            {
-                options.ConnectionMode(ConnectionMode.Gateway);
-                options.WebProxy(new WebProxy());
-                options.Region(Regions.WestEurope);
-                options.GatewayModeMaxConnectionLimit(32);
-            });
+    {
+        optionsBuilder.UseCosmos(
```

Comment:
Why change this?

Reply:
I do not remember :) Perhaps I waned to delete it and move all somwhere else and after saving changed my mind and return back.